addons/l10n_in/tests/test_tds_tcs_alert.py

```diff
@@ -80,6 +80,10 @@ def setUpClass(cls):
                 }),
             ],
         })
+        cls.env.company.child_ids.write({
+            'l10n_in_tds_feature': True,
+            'l10n_in_tcs_feature': True,
+        })
```

Comment:
When we activated this in the parent company it's auto-activated in the child company.
because taxes are shared with the child companys.

Reply:
done